test/unit_tests/braket/tasks/test_gate_model_quantum_task_result.py

```diff
@@ -373,3 +373,35 @@ def test_calculate_ir_results_value_error():
     measured_qubits = [0]
     measurements = np.array([[0]])
     GateModelQuantumTaskResult._calculate_result_types(ir_string, measurements, measured_qubits)
+
+
+def test_hash_result_types():
```

Comment:
Nice! Try using `pytest.mark.parametrize`, like what's done [here](https://github.com/aws/amazon-braket-sdk-python/blob/main/test/unit_tests/braket/circuits/test_circuit.py#L734-L760). This way, you can use the same test code and just add more cases.

Reply:
changed!